ui/src/app/pages/access/registered-tier-card.tsx

```diff
@@ -23,7 +23,7 @@ import { ModulesForInitialRegistration } from './modules-for-initial-registratio
 
 // Sep 16 hack while we work out some RAS bugs
 const TemporaryRASModule = (props: { profile: Profile }) => {
```

Comment:
Unrelated to this PR but relevant to the overall epic, could you update this comment and the name of the component? I'm unsure what the "hack" is and it appears it wasn't temporary.

Reply:
That sounds like a good idea. I'd like to look into what the RAS bugs were.

Reply:
"Sep 16" refers to 2021, haha.  Let me see what else I can find here...

https://github.com/all-of-us/workbench/pull/5605 - the original intent was to launch the new access modules page (Data Access Requirements) without RAS, because the RAS integration wasn't actually ready yet.  However, part of the reason for the new page was to showcase the new RAS integration!  So we went with this "coming soon" compromise.

Later of course we launched RAS, and this is not displayed when the FF is set to true.  But we still don't have it enabled in Preprod, and so this text is displayed there.